### PR TITLE
Copy ignore-2.12.txt to ignore-2.13.txt

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Relates to https://github.com/ansible-collections/overview/issues/45#issuecomment-926794724

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @briantist

File: .github/workflows/ansible-test.yml [hunk 1]
```diff
@@ -146,6 +146,12 @@ jobs:
         exclude:
           - ansible: stable-2.9
             python: 3.9
+          - ansible: devel
+            python:
+              - 2.7
+              - 3.5
+              - 3.6
+              - 3.7
```
Comment (briantist) [inline .github/workflows/ansible-test.yml]: I don't think you can use a list here. It seems like your checks did run devel with 2.7 for example.

Instead you need to add a separate exclude entry for every combination, unfortunately.

BTW, I will also recommend you take this time to change all your python version entries to strings by quoting them. These work fine until you try to add in `3.10` and you get errors about python 3.1.

Comment (Andersson007) [reply .github/workflows/ansible-test.yml]: good spot, will do, thank you!

**Comment by Andersson007** [comment]
@briantist @felixfontein thanks for your help!

I think I'll add 2.13 later at once. As it's needed for all the collection, will try to automate it somehow

**Pull request merged**